docs/content/en/docs/reference/vsphere/vsphere-prereq.md

```diff
@@ -8,14 +8,18 @@ description: >
 
 To run EKS Anywhere, you will need:
 
+### Prepare Administrative machine
+Set up an Administrative machine as described in [Install EKS Anywhere ]({{< relref "../../getting-started/install/" >}}).
+
+### Prepare a VMware vSphere environment
+To prepare a VMware vSphere environment to run EKS Anywhere, you need the following:
 * A vSphere 7+ environment running vCenter
 * Capacity to deploy 6-10 VMs
-* DHCP service running in vSphere environment in the primary VM network for your workload cluster
+* [DHCP service]({{< relref "./vsphere-dhcp/" >}}) running in vSphere environment in the primary VM network for your workload cluster
 * One network in vSphere to use for the cluster. This network must have inbound access into vCenter
-* A OVA imported into vSphere and converted into template for the workload VMs
-* User credentials to [create VMs and attach networks, etc]({{< relref "user-permissions.md" >}})
-* One IP address routable from cluster but excluded from DHCP offering
-
+* An [OVA]({{< relref "./vsphere-ovas/" >}}) imported into vSphere and converted into a template for the workload VMs
+* User credentials to [create VMs and attach networks, etc]({{< relref "./user-permissions/" >}})
```

Comment:
```suggestion
* User credentials to create VMs and attach networks, etc
```

Reply:
I didn't realize that user-permissions had been deleted. Hopefully this will take care of it.